packages/design-system/src/components/Label/Label.tsx

```diff
@@ -1,7 +1,8 @@
-import { Text } from '@ds/components'
-import { styled } from '@ds/stitches.config'
 import * as LabelPrimitive from '@radix-ui/react-label'
 
+import { styled } from '../../stitches.config'
+import { Text } from '../Text/Text'
```

Comment:
To be honest, I think this is because of a import/export condition when grabbing from  the root.

Please verify if this is necessary, or if we can avoid this pattern long-term.

May need the `.toString` method within?

Reply:
Hrm, that was not it. But we do not need to double `Text/Text` at least.